### PR TITLE
Update travis to use miniconda

### PR DESCRIPTION
**Pull request by daf**
Builds pass and are rapid!

See [Build 3](https://travis-ci.org/daf/pyugrid/builds/70080669)

@ChrisBarker-NOAA if you accept this PR, it should automatically add itself to #37

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,15 +1,17 @@
 language: python
-python:
-  - "2.7"
+env:
+  - CONDA="python=2.7"
```
Comment (ocefpaf) [inline .travis.yml]: Can you add Python 3.4 here as well?

File: .travis.yml [hunk 2]
```diff
@@ -1,15 +1,17 @@
...
-  - "pip install numpy"
-  - "pip install netCDF4"
-script: "py.test"
+  - URL=http://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh
+  - wget $URL -O miniconda.sh
+  - bash miniconda.sh -b -p $HOME/miniconda
+  - export PATH="$HOME/miniconda/bin:$PATH"
+  - conda update --yes conda
+  - conda config --add channels ioos --force
+  - travis_retry conda create --yes -n test --file requirements.txt $CONDA
```
Comment (ocefpaf) [inline .travis.yml]: There is a [conda-requirements.txt](https://github.com/pyugrid/pyugrid/blob/master/conda-requirements.txt).  If this is working maybe you could delete the other file in this PR.

**Comment by daf** [comment]
Woof, looks like I did some duplicate work, not realizing that this branch had diverged from master so much. See e2e8edae4db411ff0ee3.  Closing and will merge master to this branch as a diff PR.

**Comment by ChrisBarker-NOAA** [comment]
>
> Builds pass and are rapid!
>
> See Build 3 <https://travis-ci.org/daf/pyugrid/builds/70080669>
>

very cool.

It looks like it's setting up a virtualenv first, then using miniconda --
maybe I'm confused, but with conda, is there any need for a virtualenv?



> @ChrisBarker-NOAA <https://github.com/ChrisBarker-NOAA> if you accept
> this PR, it should automatically add itself to #37
> <https://github.com/pyugrid/pyugrid/pull/37>
>
Since this is a Travis Build issue -- I"M going to punt to Chris Calloway
-- does this look good to you?

-Chris B



>  ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/pyugrid/pyugrid/pull/62
> Commit Summary
>
>    - Change travis to use miniconda
>
> File Changes
>
>    - *M* .travis.yml
>    <https://github.com/pyugrid/pyugrid/pull/62/files#diff-0> (24)
>
> Patch Links:
>
>    - https://github.com/pyugrid/pyugrid/pull/62.patch
>    - https://github.com/pyugrid/pyugrid/pull/62.diff
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pyugrid/pyugrid/pull/62>.
>



-- 

Christopher Barker, Ph.D.
Oceanographer

Emergency Response Division
NOAA/NOS/OR&R            (206) 526-6959   voice
7600 Sand Point Way NE   (206) 526-6329   fax
Seattle, WA  98115       (206) 526-6317   main reception

Chris.Barker@noaa.gov

**Comment by daf** [comment]
@ChrisBarker-NOAA looks like that's travis default behavior for python projects.  I closed this because master has already done it, so I'm going to merge master into this branch and get it all up to snuff.  New PR incoming this afternoon.

**Comment by ChrisBarker-NOAA** [comment]
thanks for cleaning all this up!

-Chris


On Wed, Jul 8, 2015 at 10:29 AM, Dave Foster <notifications@github.com>
wrote:

> @ChrisBarker-NOAA <https://github.com/ChrisBarker-NOAA> looks like that's
> travis default behavior for python projects. I closed this because master
> has already done it, so I'm going to merge master into this branch and get
> it all up to snuff. New PR incoming this afternoon.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pyugrid/pyugrid/pull/62#issuecomment-119671042>.
>



-- 

Christopher Barker, Ph.D.
Oceanographer

Emergency Response Division
NOAA/NOS/OR&R            (206) 526-6959   voice
7600 Sand Point Way NE   (206) 526-6329   fax
Seattle, WA  98115       (206) 526-6317   main reception

Chris.Barker@noaa.gov

**Pull request closed without merging**